src/device-registry/routes/api-v1.js

```diff
@@ -2204,9 +2204,9 @@ router.post(
         })
         .bail()
         .customSanitizer((value) => {
-          return numeral(value).format("0.00000");
+          return numeral(value).format("0.00000000000000");
```

Comment:
Updating the input validations for Site creation. Where Sanitization of the provided GPS coordinates is now changed from 5 decimal places to 14. "The more the decimal places, the better the accuracy of the coordinates".